tests/plugins/test_plugins_manager.py

```diff
@@ -64,3 +65,32 @@ def test_app_blueprints(self):
         # Blueprint should be present in the app
         self.assertTrue('test_plugin' in self.app.blueprints)
         self.assertEqual(self.app.blueprints['test_plugin'].name, bp.name)
+
+
+@mock.patch('airflow.plugins_manager.import_errors', return_value={})
+@mock.patch('airflow.plugins_manager.plugins', return_value=[])
+@mock.patch('airflow.plugins_manager.pkg_resources.iter_entry_points')
+def test_entrypoint_plugin_errors_dont_raise_exceptions(
+    mock_ep_plugins, mock_plugins, mock_import_errors, caplog
+):
+    """
+    Test that Airflow does not raise an Error if there is any Exception because of the
+    Plugin.
+    """
+    from airflow.plugins_manager import load_entrypoint_plugins, import_errors
+
+    mock_entrypoint = mock.Mock()
+    mock_entrypoint.name = 'test-entrypoint'
+    mock_entrypoint.module_name = 'test.plugins.test_plugins_manager'
+    mock_entrypoint.load.side_effect = Exception('Version Conflict')
+    mock_ep_plugins.return_value = [mock_entrypoint]
+
+    # Clear Logs
+    caplog.clear()
```

Comment:
This causes some problems. Personally, I use `self.assertLogs`.

Reply:
I wonder why though, it is passing locally with Breeze

Reply:
I suspect that the side effect in this file:  https://github.com/apache/airflow/blob/master/tests/test_logging_config.py

Reply:
Hopefully should work now 🤞 - used self.assertLogs

Reply:
Created https://github.com/apache/airflow/pull/8746 to fix side effects from test_logging_config